src/core/server/saved_objects/import/import_saved_objects.ts

```diff
@@ -26,14 +26,17 @@ import {
 } from './types';
 import { validateReferences } from './validate_references';
 
-export async function importSavedObjects({
-  readStream,
-  objectLimit,
-  overwrite,
-  savedObjectsClient,
-  supportedTypes,
-  namespace,
-}: SavedObjectsImportOptions): Promise<SavedObjectsImportResponse> {
+export async function importSavedObjects(
+  {
+    readStream,
+    objectLimit,
+    overwrite,
+    savedObjectsClient,
+    supportedTypes,
+    namespace,
+  }: SavedObjectsImportOptions,
+  sourceSpaceId?: string
```

Comment:
We shouldn't need to pass this additional `sourceSpaceId` string. The `namespace` property on the options is used to denote the target namespace/spaceId.

Reply:
sourceSpaceId is needed to extract the correct title for the objects we are about to overwrite.